### PR TITLE
Add hostname to etcd backup folder name

### PR DESCRIPTION
**Pull request by jhadvig**
@ingvagabund PTAL

fixes https://github.com/openshift/openshift-ansible/issues/4475

### CODE REVIEW

File: playbooks/common/openshift-cluster/upgrades/etcd/backup.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@
     r_etcd_common_backup_tag: etcd_backup_tag
     r_etcd_common_etcd_runtime: "{{ openshift.common.etcd_runtime }}"
     r_etcd_common_embedded_etcd: "{{ groups.oo_etcd_to_config | default([]) | length == 0 }}"
-    r_etcd_common_backup_sufix_name: "{{ lookup('pipe', 'date +%Y%m%d%H%M%S') }}"
+    r_etcd_common_backup_sufix_name: "{{ inventory_hostname + '-' + lookup('pipe', 'date +%Y%m%d%H%M%S') }}"
```
Comment (ingvagabund) [inline playbooks/common/openshift-cluster/upgrades/etcd/backup.yml]: @sdodson WDYT? Do we have any naming schema for etcd backups?

**Comment by ingvagabund** [comment]
aos-ci-test

**Comment by sdodson** [comment]
I'd still like to understand why this is necessary. /var/lib/etcd should never be mounted across hosts so there should never be any conflict.

**Comment by jhadvig** [comment]
After some investigation I found out that the issue was cause by launching the  upgrade playbook with `--connection local` flag. After removing it the issue cant be reproduced. Closing this one.

**Comment by sdodson** [comment]
now it makes sense! thanks

**Pull request closed without merging**